Brooklyn/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,70 @@
+/*
+  Localized.strings
+  Brooklyn
+
+  Created by Michael Charland on 2019-10-07.
+  Copyright © 2019 Pedro Carrasco. All rights reserved.
+*/
+
+"original" = "Apple's Original Version";
```

Comment:
Still not my cup of tea as it doesn't provide context.
You could add the prefix ´animation´ -> ´animation.original´.
Then it would become:
```swift
extension Animation {

    var name: String {
        return "animation.\(rawValue)".localized
    }
}
```

What do you think?

Reply:
Did you forget this? Also, the rest of the translations should also have a prefix in my opinion, such as ´menu.` and `preference.`

Reply:
I agree for animation that is makes sense to have it for extra information. But for menu and preference I feel normally that's it too specific. At the localizable level you normally want to be able to reuse the strings. But in this case the strings seem pretty unique per set. So I've gone ahead and added the changes.

Reply:
I kinda of prefer being specific and then generalise if needed 👍 
Thanks for everything @mike011 and welcome to the list of contributors! ❤️